modules/appeals_api/app/services/appeals_api/pdf_construction/generator.rb

```diff
@@ -52,6 +55,15 @@ def insert_additional_pages
         combine_form_fill_and_additional_pages(additional_pages_added_path)
       end
 
+      def rebuild_pages
```

Comment:
Is this rebuilding the completed PDF, or reordering it? Is the reordering specific to NOD, or is this going to be a final pass through step for every PDF?

Reply:
Was having trouble naming this because I wanted to try to get across that rebuilding could be a rearranging of the pages and/or a removal of pages, since that's come up a few times. I was thinking this would be a step for all PDFs, even if (like with add_additional_pages) the step is skipped by returning the original path for forms where it's not needed.

Reply:
format_pages?